tests/validation/converters/test_pandapower_converter_output.py

```diff
@@ -126,6 +127,88 @@ def test_generate_output_3ph():  # TODO: REMOVE THIS FUNCTION
+            np.maximum(i_a_hv / i_max_hv, i_a_lv / i_max_lv) * 100, net.res_trafo_3ph.loading_a_percent
+        )
+        np.testing.assert_allclose(
+            np.maximum(i_b_hv / i_max_hv, i_b_lv / i_max_lv) * 100, net.res_trafo_3ph.loading_b_percent
+        )
+        np.testing.assert_allclose(
+            np.maximum(i_c_hv / i_max_hv, i_c_lv / i_max_lv) * 100, net.res_trafo_3ph.loading_c_percent
+        )
+        np.testing.assert_allclose(
+            np.maximum(
+                np.maximum(net.res_trafo_3ph.loading_a_percent, net.res_trafo_3ph.loading_b_percent),
+                net.res_trafo_3ph.loading_c_percent,
+            ),
+            net.res_trafo_3ph.loading_percent,
+        )
+        np.testing.assert_allclose(
+            np.maximum(
+                np.maximum(net.res_line_3ph.loading_a_percent, net.res_line_3ph.loading_b_percent),
+                net.res_line_3ph.loading_c_percent,
+            ),
+            net.res_line_3ph.loading_percent,
+        )
+
+    def compare_result(actual, expected, *, rtol):
+        np.testing.assert_allclose(actual.trafo.vn_hv_kv, expected.trafo.vn_hv_kv, rtol=rtol)
+        np.testing.assert_allclose(actual.trafo.vn_lv_kv, expected.trafo.vn_lv_kv, rtol=rtol)
+        np.testing.assert_allclose(actual.trafo.sn_mva, expected.trafo.sn_mva, rtol=rtol)
+        np.testing.assert_allclose(
+            actual.res_trafo_3ph.loc[:, "i_a_hv_ka"], expected.res_trafo_3ph.loc[:, "i_a_hv_ka"], rtol=rtol
+        )
+        np.testing.assert_allclose(
+            actual.res_trafo_3ph.loc[:, "i_b_hv_ka"], expected.res_trafo_3ph.loc[:, "i_b_hv_ka"], rtol=rtol
+        )
+        np.testing.assert_allclose(
+            actual.res_trafo_3ph.loc[:, "i_c_hv_ka"], expected.res_trafo_3ph.loc[:, "i_c_hv_ka"], rtol=rtol
+        )
+        np.testing.assert_allclose(
+            actual.res_trafo_3ph.loc[:, "i_a_lv_ka"], expected.res_trafo_3ph.loc[:, "i_a_lv_ka"], rtol=rtol
+        )
+        np.testing.assert_allclose(
+            actual.res_trafo_3ph.loc[:, "i_b_lv_ka"], expected.res_trafo_3ph.loc[:, "i_b_lv_ka"], rtol=rtol
+        )
+        np.testing.assert_allclose(
+            actual.res_trafo_3ph.loc[:, "i_c_lv_ka"], expected.res_trafo_3ph.loc[:, "i_c_lv_ka"], rtol=rtol
+        )
+        np.testing.assert_allclose(
+            actual.res_trafo_3ph.loading_a_percent, expected.res_trafo_3ph.loading_a_percent, rtol=rtol
+        )
+        np.testing.assert_allclose(
+            actual.res_trafo_3ph.loading_b_percent, expected.res_trafo_3ph.loading_b_percent, rtol=rtol
+        )
+        np.testing.assert_allclose(
+            actual.res_trafo_3ph.loading_c_percent, expected.res_trafo_3ph.loading_c_percent, rtol=rtol
+        )
+
+    pgm_net = pp_net_3ph_minimal_trafo()
+    pp_net = pp_net_3ph_minimal_trafo()
+    pp.runpp_pgm(pgm_net, symmetric=False)
+    pp.runpp_3ph(pp_net)
+    check_result(pgm_net)
+    check_result(pp_net)
+    compare_result(pgm_net, pp_net, rtol=0.004)
```

Comment:
can you maybe add 2 test cases?

1. balanced => `rtol=0.004`
2. unbalanced => `rtol=0.04` and also add a check that the test fails when `rtol=0.004`, as well as a comment that explains why (namely that pandapower does not model the unbalanced case correctly)

Reply:
resolved